doc/index_scoped_tiered_access.md

```diff
@@ -0,0 +1,41 @@
+# Index-scoped Tiered-Access
```

Comment:
This doc is good. But I still think you should update the `README.md` as well, especially for these 2 sections https://github.com/uc-cdis/guppy#configurations and https://github.com/uc-cdis/guppy#tiered-access because they will show up in the root page of this repo